src/gui/transferlistmodel.cpp

```diff
@@ -59,6 +60,42 @@ static QIcon getErrorIcon();
 
 static bool isDarkTheme();
 
+namespace
+{
+    QHash<BitTorrent::TorrentState, QColor> torrentStateForegroundColorsFromUITheme()
+    {
+        const QPair<BitTorrent::TorrentState, QString> colorIds[] =
```

Comment:
Let's keep all these "pairs" for using in abstract algorithms. I would prefer to see appropriate structures in all possible cases.